src/sections.js

```diff
@@ -88,6 +80,23 @@ const splitStringWithArguments = (string) => {
     return result;
 };
 
+/**
+ * Set path argument. This is an argument, which used for analyzing
+ * directory/files with CLI w/o additional commands and arguments
+ * Example: $ eslint ~/js/my_project
+ * @param {object} context - internal config
+ * @return {object} argument - path argument
+ */
+const setPathArgument = (context) => {
+    let argument = context.get.template.argument();
```

Comment:
We can declare properties inside the object and return it

Reply:
@svoboda-rabstvo done.